leveldb_vendor/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>leveldb_vendor</name>
+  <version>0.0.1</version>
+  <description>leveldb vendor package, providing a dependency for leveldb.</description>
+  <maintainer email="barry.xu@sony.com">Barry Xu</maintainer>
+  <license>Apache License 2.0</license>  <!-- the contents of this package are Apache 2.0 -->
+  <license>BSD</license>  <!-- leveldb is BSD -->
+
+  <url type="website">https://github.com/google/leveldb</url>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>snappy</depend>
```

Comment:
is `snappy` available on all platforms or does this have to be vendor`ed as well?

Reply:
Yeah.  I am considering this.